src/sentry/utils/snql.py

```diff
@@ -54,25 +54,29 @@ def get_option(self, referrer: Optional[str]) -> Optional[SNQLOption]:
     },
     allowlist=set(),
     prefixes=[
-        "tsdb-modelid:",
         "tagstore.",
         "group.",
         "search",
         "serializers.",
         "eventstore.",
         "search_sample",
         "testing.test",
-        "api.",
         "discover",
     ],
     by_entity={},
 )
 
 snql_check = ReferrerCheck(
     option="snuba.snql.snql_only",
-    denylist=set(),
+    denylist={
+        "tsdb-modelid:4",
+        "tsdb-modelid:300",
+        "tsdb-modelid:200",
+        "tsdb-modelid:202",
+        "tsdb-modelid:100",
```

Comment:
TSDB is used a bit in usage calculation and spike protection calculations as well. The queries in those spots are pretty so they likely won't be a problem.